tests/integ/test_packaging.py

```diff
@@ -539,3 +615,20 @@ def plus_one_month(x):
     Utils.check_answer(
         df.select(plus_one_month_udf("a")).collect(), [Row(plus_one_month(d))]
     )
+
+
+def test_get_available_versions_for_packages(session):
+    """
+    Assert that the utility function get_available_versions_for_packages() returns a list of versions available in Snowflake,
+    for some common packages.
+    """
+    package_table = "information_schema.packages"
+    # TODO: Use the database from fully qualified UDF name
+    if not session.get_current_database():
```

Comment:
do you want to take care of the todo?

Reply:
Not sure what the TODO is about, it was in the codebase prior to my changes.